packages/visualizations/src/components/Map/Combinations/NavigableMap.svelte

```diff
@@ -0,0 +1,85 @@
+<script lang="ts">
+    import type { BBox } from 'geojson';
+    import VectorChoropleth from '../WebGl/ChoroplethVectorTiles.svelte';
+    import type { ChoroplethDataValue, NavigableChoroplethOptions, NavigationMap } from '../types';
+
+    import BackButton from './BackButton.svelte';
+    import MiniMap from './MiniMap.svelte';
+
+    export let data: { value: ChoroplethDataValue[] };
+    export let options: NavigableChoroplethOptions;
+
+    $: ({ navigationMaps, colorScale, bbox, ...mainOptions } = options);
+
+    const defaultInteractive = true;
+    $: ({ interactive = defaultInteractive } = options);
+
+    let viewBox: BBox | undefined;
+    $: viewBox = bbox;
+
+    let active: number | undefined;
+    let height: number;
+
+    const setViewBox = (map: NavigationMap, i: number) => () => {
```

Comment:
The `interactive` prop should also block clicks on the button. Ideally it should also remove the hover effect as well... Worst case, a good old `pointer-events: none;` would block all that, although a bit roughly (and imperfectly in terms of accessibility)